APIs/openEO/Python_Client/Python.qmd

```diff
@@ -0,0 +1,346 @@
+The most common or popular openEO processes have a dedicated `Datacube` method (e.g. `mask`, `aggregate_spatial`, `filter_bbox`, ...). 
+Other processes without a dedicated method can still be applied in a generic way.
+An on top of that, there are also some convenience methods that implement
+openEO processes is a compact, Pythonic interface.
+
+For example, the [`min_time`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.datacube.DataCube.min_time){target="_blank"} method
+implements a `reduce_dimension` process along the temporal dimension, using the `min` process as reducer function:
+
+```python
+datacube = datacube.min_time()
+```
+
+This creates a new datacube (we overwrite the existing variable),
+where the time dimension is eliminated and for each pixel we just have 
+the minimum value of the corresponding timeseries in the original datacube.
+
+See the [Python client `Datacube` API](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.datacube.DataCube){target="_blank"} for a more complete listing of methods that implement openEO processes.
+
+
+openEO processes that are not supported by a dedicated `Datacube` method
+can be applied in a generic way with the [`process` method](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.datacube.DataCube.process){target="_blank"}, e.g.:
+
+```python
+datacube = datacube.process(
+    process_id="ndvi", 
+    arguments={
+        "data": datacube, 
+        "nir": "B8", 
+        "red": "B4"}
+)
+```
+
+This applies the [`ndvi` process](https://docs.openeo.cloud/processes/#ndvi){target="_blank"} to the datacube with the arguments of "data", "nir" and "red" (This example assumes a datacube with bands `B8` and `B4`).
+
+
+::: {.callout-note}
+
+Still unsure on how to make use of processes with the Python client? 
+Visit the [official documentation on working with processes](https://open-eo.github.io/openeo-python-client/processes.html#working-with-processes){target="_blank"}.
+
+:::
+
+
+### Defining output format
+
+After applying all processes you want to execute, we need to tell the back-end to export the datacube, for example as GeoTiff:
+
+```python
+result = datacube.save_result("GTiff")
+```
+
+
+## Execution
+
+It's important to note that all the datacube processes we applied up to this point
+are not actually executed yet, neither locally nor remotely on the back-end.
+We just built an abstract representation of the algorithm (input data and processing chain), 
+encapsulated in a local `Datacube` object (e.g. the `result` variable above).
+To trigger an actual execution (on the back-end) we have to explicitly send this representation 
+to the back-end.
+
+openEO defines [several processing modes](../Glossary.qmd#data-processing-modes), 
+but for this introduction we'll focus on batch jobs, which is a good default choice.
```

Comment:
batch jobs are not a good default choice. In fact, the default should be synchronous jobs on small areas because they are much faster.

Reply:
removed that section